packages/p2p-v2/src/components/BuySellForm/BuySellForm.scss

```diff
@@ -9,6 +9,10 @@
 
     & .deriv-modal__body {
         padding: 0;
+        @include mobile {
+            height: calc(100% - 14rem);
+            overflow-y: scroll;
+        }
```

Comment:
isn't it already scrollable?

Reply:
ah i need to remove this, cause last time the form wasnt scrollable, this was before we merged your card